exotica_python/src/Exotica.cpp

```diff
@@ -552,6 +553,19 @@ PYBIND11_MODULE(_pyexotica, module)
     samplingProblem.def("getSpaceDim", &SamplingProblem::getSpaceDim);
     samplingProblem.def("getBounds", &SamplingProblem::getBounds);
 
+    py::class_<CollisionProxy, std::shared_ptr<CollisionProxy>> proxy(module, "Proxy");
+    proxy.def(py::init());
+    proxy.def_readonly("Contact1", &CollisionProxy::contact1);
```

Comment:
A bit at unease with the inconsistency of using Uppercase at the start of variables - sometimes it's lower case (most common), then in other places suddenly uppercase...